dist/index.html

```diff
@@ -12,7 +12,9 @@
     <h1 class = "title">What's Cookin?</h1>
     <nav class="navbar">
       <button class="all-recipes">All Recipes</button>
-      <input id = "search-bar" type="text" placeholder=" Search...">
+      <input class = "nav-search-bar" id = "search-bar" type="text" placeholder=" Search...">
+      <input class = "saved-search hidden" id = "saved-search-bar" type="text" placeholder=" Search Saved Recipes...">
```

Comment:
I love that the user can search through their savedRecipes without looking for another search bar!